pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "itwinai"
 version = "0.1"
 description = "AI and ML workflows module for interTwin"
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.8"
```

Comment:
is python 3.8 needed?

Reply:
3.9 is not available for the NGC container